pom.xml

```diff
@@ -25,10 +25,10 @@
   <properties>
     <jdk.version>11</jdk.version>
 
-    <atoti-server.version>6.0.10</atoti-server.version>
-    <atoti-ui.version>5.1.13</atoti-ui.version>
+    <atoti-server.version>6.1.3-SNAPSHOT</atoti-server.version>
+    <atoti-ui.version>5.2.0-beta</atoti-ui.version>
```

Comment:
You should not have to set the UI version anymore. Using the starters, we remove this need